gcp-cloud-run/variables.tf

```diff
@@ -0,0 +1,272 @@
+#################
+variable "vault_version" {
+  description = "Vault version"
+  type        = string
+  default     = "1.7.1"
+}
+variable "vault_region" {
+  description = "Vault Region. if empty fallback to default region"
+  type        = string
+  default     = null
+}
+
+variable "image" {
+  description = "Vault server image name"
+  type        = string
+  default     = "us-central1-docker.pkg.dev/piiano/docker/pvault-server"
+}
+
+
+variable "vault_license" {
+  description = "Vault server license token"
+  type        = string
+  default     = null
+}
+
+variable "vault_sql_serverless_connector_range" {
+  description = "Cloud Run connector /28 CIDR range (used to connect Cloud Run to VPC)"
+  type        = string
+  default     = "10.8.0.0/28"
+}
+
+variable "connector_cloud_run_max_instances" {
+  description = "Maximum number of instances used by VPC Serverless connector"
+  type        = number
+  default     = 4
+}
+
+########################
+### Vault CLI Server ###
+########################
+variable "vault_cli_subnet" {
+  description = "Subnet where Vault CLI will be deployed"
+  type        = string
+  default     = "sb-vault-authorized"
+}
+
+variable "vault_cli_subnet_range" {
+  description = "Subnet CIDR range for the Vault CLI VM"
+  type        = string
+  default     = "10.8.0.16/28"
+}
+
+variable "vault_cli_zone" {
+  description = "Zone where Vault CLI will be deployed"
+  type        = string
+  default     = null
+}
+
+variable "cli_image" {
+  description = "Vault server image name"
+  type        = string
+  default     = "us-central1-docker.pkg.dev/piiano/docker/pvault-cli"
+}
+
+###########
+### KMS ###
+###########
+variable "kms_ring_name" {
+  description = "KMS Ring name"
+  type        = string
+  default     = "key-ring"
+}
+
+variable "vault_kms_key_name" {
+  description = "KMS key name"
+  type        = string
+  default     = "vault-key"
+}
+
+variable "db_kms_key_name" {
+  description = "KMS key name"
+  type        = string
+  default     = "db-key"
+}
+
+#################
+### Cloud SQL ###
+#################
+variable "db_instance_name" {
+  description = "Database instance name"
+  type        = string
+  default     = "vault-sql-db"
+}
+
+variable "db_name" {
+  description = "Vault database name"
+  type        = string
+  default     = "pvault"
+}
+
+variable "db_version" {
+  description = "Postgres database version"
+  type        = string
+  default     = "POSTGRES_14"
+}
+
+variable "db_zone" {
+  description = "Database zone. if empty fallback to default zone"
+  type        = string
+  default     = null
+}
+
+variable "db_region" {
+  description = "Database region. if empty fallback to default region"
+  type        = string
+  default     = null
+}
+
+variable "db_user" {
+  description = "Vault database user"
+  type        = string
+  default     = "pvault"
+}
+
+variable "db_tier" {
+  description = "Database instance tier"
+  type        = string
+  default     = "db-f1-micro"
```

Comment:
@david-piiano is that a reasonable default?

Reply:
For trial purposes - yes. I will make sure to highlight it in the readme that for production or any amount of significant testing you require a stronger one